sdk/core/Azure.Core.TestFramework/src/testproxy.json

```diff
@@ -517,15 +517,15 @@
     "CustomDefaultMatcher": {
```

Comment:
Move this file to test-proxy repo?

Reply:
The swagger will be automated in the test-proxy repo in the future.